src/main/java/VASSAL/build/widget/HtmlChart.java

```diff
@@ -129,6 +131,10 @@ public Component getComponent() {
       scroller = new ScrollPane(htmlWin);
       scroller.getViewport().setPreferredSize(htmlWin.getPreferredSize());
       scroller.getViewport().setAlignmentY(0.0F);
+      
+      final Font f = new JLabel().getFont();
```

Comment:
I wonder if there's a way to do this without creating a `JLabel` which is never displayed. I don't want to hold up the PR for that, but it's something to think about.